x/stakeibc/keeper/msg_server_redeem_stake_test.go

```diff
@@ -92,7 +93,7 @@ func (suite *KeeperTestSuite) TestRedeemStakeSuccessful() {
 
 	msg := tc.validMsg
 	user := tc.user
-	redeemAmount := sdk.NewInt(msg.Amount)
+	redeemAmount := sdk.NewInt(cast.ToInt64(msg.Amount))
```

Comment:
flagging this

Reply:
Addressed per new cast fork and using cast.To____E